playbook/app/pb_kits/playbook/pb_distribution_bar/_distribution_bar.jsx

```diff
@@ -6,6 +6,7 @@ import { globalProps } from '../utilities/globalProps.js'
 
 type DistributionBarProps = {
   className?: string,
+  colors: array,
```

Comment:
```suggestion
  colors: [string],
```

Reply:
This way it's more specific what you expect in the array type wise.

Reply:
Looks like in flowjs the bracket `[type]` notation is for [tuples of pre-determined length](https://flow.org/en/docs/types/tuples/), [arrays of indeterminate/variable length](https://flow.org/en/docs/types/arrays/) use a template style notation: `Array<string>`.

That said none of these types are strictly enforced.  Don't think we even get warnings out of them.  So I guess top priority is human readability.